cmd/dbseed/candle.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/thrasher-corp/gocryptotrader/database/repository/candle"
+	"github.com/urfave/cli/v2"
+)
+
+var seedCandleCommand = &cli.Command{
+	Name:  "candle",
+	Usage: "seed candle data",
+	Subcommands: []*cli.Command{
+		{
+			Name:  "file",
+			Usage: "seed candle data from a file",
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:  "exchange",
+					Usage: "Exchange name of supplied candle data",
+				},
+				&cli.StringFlag{
+					Name:  "base",
+					Usage: "Base currency of supplied candle data",
+				},
+				&cli.StringFlag{
+					Name:  "quote",
+					Usage: "Quote currency of supplied candle data",
+				},
+				&cli.StringFlag{
```

Comment:
I think it would be good to provide the examples formats here. I went with 86400, but discovered that 24h is the standard

Reply:
I have updated this command (and the database) to store in second format to match what we pass into the gethistoriccandle methods 

Also provided a migration to update existing records